core/message-tags-3.3.md

```diff
@@ -80,6 +89,24 @@ Individual tag keys MUST only be used a maximum of once per message. Clients
 receiving messages with more than one occurrence of a tag key SHOULD discard all
 but the final occurrence.
 
+### Empty tagged messages
+
+Servers MUST accept `PRIVMSG` and `NOTICE` events sent with client-only tags even if
+the message content parameter is omitted. These events MUST NOT be delivered to clients
+that haven't negotiated the message tags capability. If neither client-only tags nor a
+message parameter are included, servers MAY respond with a standard `ERR_NEEDMOREPARAMS`
+error.
+
+Clients that receive events without a message text MUST NOT display empty messages as-is
+in the message history unless they are using the attached tags according to their own
```

Comment:
But... how else can they use those tags? According to specs of some other unrelated tags?